cmd/sovereignnode/chainSimulator/configs/configs_test.go

```diff
@@ -0,0 +1,27 @@
+package configs
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/multiversx/mx-chain-go/node/chainSimulator/configs"
+)
+
+func TestGenerateSovereignGenesisFile(t *testing.T) {
+	if testing.Short() {
+		t.Skip("this is not a short test")
+	}
+
+	outputConfig, err := configs.CreateChainSimulatorConfigs(configs.ArgsChainSimulatorConfigs{
```

Comment:
You copy-pasted the test and instead of testing `GenerateSovereignGenesisFile`, you are testing `CreateChainSimulatorConfigs`...

Reply:
Actually, you are testing `CreateChainSimulatorConfigs` func, not the `GenerateSovereignGenesisFile` func itself